pkgs/development/python-modules/openusd/default.nix

```diff
@@ -65,13 +66,11 @@ buildPythonPackage rec {
 
   outputs = [ "out" ] ++ lib.optional withDocs "doc";
 
-  format = "other";
-
   patches = [
     (fetchpatch {
       name = "port-to-embree-4.patch";
-      url = "https://github.com/PixarAnimationStudios/OpenUSD/pull/2266/commits/4b6c23d459c602fdac5e0ebc9b7722cbd5475e86.patch";
-      hash = "sha256-yjqdGAVqfEsOX1W/tG6c+GgQLYya5U9xgUe/sNIuDbw=";
+      url = "https://github.com/PixarAnimationStudios/OpenUSD/pull/2266/commits/c8fec1342e05dca98a1afd4ea93c7a5f0b41e25b.patch";
```

Comment:
done

Reply:
@amarshall this is the first I hear of this parameter. Could you elaborate? Is this something we always want? If so, we should make it conventional or even required (via RFC) and perhaps make `fetchpatch` apply it automatically.

Reply:
`full_index=1` uses full commit hashes instead of shortened ones. I don't think it matters too much in this case, as we aren't applying patches using `git apply`/`git am`